react-ui/src/components/Layouts/InputWorkerTimes/CheckboxApplication.js

```diff
@@ -0,0 +1,135 @@
+import React  from 'react';
+import { render } from 'react-dom';
+import './CheckboxApplication.css';
+import {Container, Row, Col} from 'react-grid-system';
+import { Redirect, Route, withRouter, Link, Router } from "react-router-dom";
+
+import moment from 'moment'
+       
+class Check extends React.Component { // this is a class component for the checkbox feature of selecting recurring days
+  constructor() {
+    super();
+    this.state = {
+      Days: [
+        {id: 1, value: "Monday"},
+        {id: 2, value: "Tuesday"},
+        {id: 3, value: "Wednesday"},
+        {id: 4, value: "Thursday"},
+        {id: 5, value: "Friday"},
+        {id: 6, value: "Saturday"},
+        {id: 7, value: "Sunday"}
+      ],
+      checkedDays: new Map()
+    };
+  
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  
+  daysInMonth = () => {
+    return this.state.dateContext.daysInMonth();
+ }
+
+ firstDayofMonth = () => {
+  let dateContext = this.state.dateContext;//calling in current dateContext, then passing it to get first day of month
+  let firstDay = moment(dateContext).startOf('month').format('d') // day of week 0 ...1..2..6, 0 is sunday, 6 is saturday
+  return firstDay;
+}
+
+     
+  handleChange(event) {
+        var isChecked = event.target.checked;
+        var item = event.target.value;
+         
+        this.setState(prevState => ({ checkedDays: prevState.checkedDays.set(item, isChecked) }));
+  }
+     
+  handleSubmit(event) { // submits selected days and assign recurrenting dates based off boxes checked
+    /* console.log(this.state);  */ //test line
```

Comment:
Hi Kyle,
Not sure if these test lines should be kept? Will they be used in the future or can they be removed entirely?

Reply:
i was just using them for debugging purposes, they can be removed entirely